public/app/themes/justice/inc/block-editor.php

```diff
@@ -91,29 +91,37 @@ public function search(): string
     }
 
     /**
-     * Format the moj-anchor anchor.
+     * Format the moj-anchor anchor and footnotes blocks.
      *
+     * moj-anchor
      * This is rich text formatting that is added to the block editor via a JS.
      * In the editor, a single space is used for compatibility, and so that the anchor can be selected.
      * The space is removed when the content is rendered, so as not to break the layout.
      *
+     * core/footnotes
+     * Edit the unicode character ↩︎ to "Back to text", to match the legacy content.
+     *
      * @param string $content
      * @return string
      */
 
-    public function formatMojAnchor($content)
+    public function formatBlocks($content)
     {
 
         // Match all a tags class of moj-anchor and a single space.
-        $pattern = '/(<a[^>]*class="moj-anchor"[^>]*>) (<\/a>)/';
+        $moj_anchor_pattern = '/(<a[^>]*class="moj-anchor"[^>]*>) (<\/a>)/';
         // Remove the single space.
-        $replacement = '$1$2';
-        $content = preg_replace($pattern, $replacement, $content);
+        $moj_anchor_replacement = '$1$2';
+        $content = preg_replace($moj_anchor_pattern, $moj_anchor_replacement, $content);
```

Comment:
In the interest of saving bytes, could this just be the following? I understand if you're using comments to explain the rationale.

```php
$content = preg_replace($moj_anchor_pattern, '$1$2', $content);
```

Reply:
Yes, that's a good point!